Code/Tools/FBuild/FBuildCore/WorkerPool/JobQueue.cpp

```diff
@@ -546,7 +548,10 @@ void JobQueue::MainThreadWait( uint32_t maxWaitMS )
 void JobQueue::WorkerThreadWait( uint32_t maxWaitMS )
 {
     ASSERT( Thread::IsMainThread() == false );
-    ASSERT( FBuild::Get().GetOptions().m_NumWorkerThreads > 0 );
+    if (FBuild::IsValid())
+    {
+        ASSERT( FBuild::Get().GetOptions().m_NumWorkerThreads > 0 );
+    }
```

Comment:
JobQueue should never exist without the FBuild singleton existing. Did you experience a crash here? If so, that would point to a problem elsewhere that needs fixing?.

Reply:
Doesn't seem to be a problem now. Disregard?